clang_tools/install.py

```diff
@@ -76,12 +84,12 @@ def move_and_chmod_binary(old_file_name, new_file_name, directory) -> None:
     try:
         if not os.path.isdir(install_dir):
             os.makedirs(install_dir)
-        shutil.move(old_file_name, f"{install_dir}/{new_file_name}")
-        os.chmod(os.path.join(install_dir, new_file_name), 0o755)
+        shutil.move(old_bin_name, f"{install_dir}/{new_bin_name}")
+        os.chmod(os.path.join(install_dir, new_bin_name), 0o755)
     except PermissionError:
         raise SystemExit(
-            f"Don't have permission to install {new_file_name} to {install_dir}. \
-                Try to run with the appropriate permissions."
+            f"""Don't have permission to install {new_bin_name} to {install_dir}.
+            Try to run with the appropriate permissions."""
```

Comment:
The use of `"""` here is what causes the huge whitespace in the error message.

It would have been better to use
```py
            f"Don't have permission to install {new_bin_name} to {install_dir}."
            + " Try to run with the appropriate permissions."
```

Reply:
Thank you :)

Reply:
Updated

Reply:
I've cloned and started re-writing some of this. I hope you don't mind. I found a bunch of duplicate code and typing problems... I'm also adding docstrings as I go.

Reply:
Re-writing is very welcome, I want the code to be better 💯